test/paranoia_test.rb

```diff
@@ -758,6 +763,60 @@ def test_missing_restore_recursive_on_polymorphic_has_one_association
     assert_equal 0, polymorphic.class.count
   end
 
+  def test_counter_cache_column_update_on_destroy#_and_restore_and_really_destroy
+    parent_model_with_counter_cache_column = ParentModelWithCounterCacheColumn.create
+    related_model = parent_model_with_counter_cache_column.related_models.create
+
+    assert_equal 1, parent_model_with_counter_cache_column.reload.related_models_count
+    related_model.destroy
+    assert_equal 0, parent_model_with_counter_cache_column.reload.related_models_count
+    # related_model.restore
+    # assert_equal 1, parent_model_with_counter_cache_column.reload.related_models_count
+    # related_model.really_destroy!
+    # assert_equal 0, parent_model_with_counter_cache_column.reload.related_models_count
+  end
+
+  def test_callbacks_for_counter_cache_column_update_on_destroy
+    parent_model_with_counter_cache_column = ParentModelWithCounterCacheColumn.create
+    related_model = parent_model_with_counter_cache_column.related_models.create
+
+    assert_equal nil, related_model.instance_variable_get(:@after_destroy_callback_called)
+    assert_equal nil, related_model.instance_variable_get(:@after_commit_on_destroy_callback_called)
+
+    related_model.destroy
+
+    assert related_model.instance_variable_get(:@after_destroy_callback_called)
+    # assert related_model.instance_variable_get(:@after_commit_on_destroy_callback_called)
```

Comment:
As I wrote [here](https://github.com/radar/paranoia/pull/192#issuecomment-74651563):
>You may want to fix the last commented test.

I don't know why it stopped working with the new changes introduced during last 2 months. Your help would be appreciated.